src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java

```diff
@@ -46,7 +46,7 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
   }
 
   private void createTitleActions(@NotNull List<? super AnAction> titleActions) {
-    AnAction action = ActionManager.getInstance().getAction("CodyChatActionsGroup");
```

Comment:
before it was an action group, now it's an action - does it still work?

Reply:
I tested it before changing - it work just fine :)